prime-router/src/main/kotlin/common/AzureHttpUtils.kt

```diff
@@ -0,0 +1,16 @@
+package gov.cdc.prime.router.common
+
+import com.microsoft.azure.functions.HttpRequestMessage
+import gov.cdc.prime.router.azure.db.Tables
+
+object AzureHttpUtils {
```

Comment:
This doesn't seem like "AzureHttpUtils" functionality

Reply:
As discussed on a huddle, this is specific to extracting the IP from an azure HttpRequest